docs/v1/P4Runtime-Spec.mdk

```diff
@@ -724,32 +724,35 @@ follows:
   time.
 
 * The streaming channel between the controller and the server defines the
-  liveness of the controller session. The controller is considered "offline" or
-  "dead" as soon as its stream channel to the switch is broken. When a primary
-  channel gets broken: (i) an advisory message is sent to all other controllers
-  for that `device_id` and `role`, as described in the
-  [following section](#sec-arbitration-notification) (ii) the P4Runtime server
-  will be without a primary controller, until a client sends a successful
-  `MasterArbitrationUpdate` (as per the rules in a
-  [later section](#sec-arbitration-updates)).
-
-* The mechanism via which the controller receives the P4Runtime server details
-  which includes the `device_id`, `ip` and `port`, as well as the mechanism via
-  which it receives the Forwarding Pipeline Config, are implementation specific
-  and beyond the scope of this specification. Similarly, the mechanism via which
-  the P4Runtime server receives its switch config (which notably includes the
-  `device_id`) is beyond the scope of this specification.  Nevertheless, if the
-  server details or switch config are transferred via the network, it is
+  liveness of the controller session. The controller is considered "offline",
+  "disconnected", or "dead" as soon as its stream channel to the switch is
+  broken. When a primary channel gets broken:
+
+  1. An advisory message is sent to all other controllers for that `device_id`
+     and `role`, as described in a
+     [later section](#sec-arbitration-notification); and
+
+  2. The P4Runtime server will be without a primary controller, until a client
+     sends a successful `MasterArbitrationUpdate` (as per the rules in a
+     [later section](#sec-arbitration-updates)).
+
+* The mechanism through which the controller receives the P4Runtime server
+  details are implementation specific and beyond the scope of this
+  specification. This includes includes the `device_id`, `ip` and `port`, as
```

Comment:
typo: extra "includes"

Reply:
Alas, you merged without me fixing the typo. Is there an easy way to just quickly do this, or should I submit another PR?

Reply:
Apologies, I must have been distracted. Please fix it in a separate PR or even in your other pending PR

Reply:
Done! Thanks!